README.md

```diff
@@ -1,2 +1,4 @@
-# set-the-minimum-and-maximum-date-for-the-date-editor-in-.NET-MAUI-DataForm
-This repository contains a sample of how to set the minimum and maximum date for the date editor in .NET MAUI DataForm (SfDataForm)?
+# How to set the minimum and maximum date for the date editor in .NET MAUI DataForm (SfDataForm)
+This repository contains a sample of how to set the minimum and maximum date for the date editor in [.NET MAUI DataForm (SfDataForm)](https://help.syncfusion.com/maui/dataform/getting-started) control.
```

Comment:
The readme file is not updated with proper details.

Reference: https://github.com/SyncfusionExamples/get-the-month-and-year-of-month-view-in-the-.NET-MAUI-Calendar/blob/master/README.md

Reply:
Updated as proposed.